src/v0/util/facebookUtils/index.test.js

```diff
@@ -639,3 +640,25 @@ describe('getContentType', () => {
     expect(result).toBe(defaultValue);
   });
 });
+
+describe('isHtmlFormat', () => {
```

Comment:
Can we add a test-case for undefined as well ?
`isHtmlFormat(undefined)`

Reply:
added few extra TC as well